app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -259,14 +275,46 @@ class ExposureNotificationsRepository @Inject constructor(
     }
 
     suspend fun checkExposure(context: Context) {
-        val lastExposure = getLastRiskyExposure()?.daysSinceEpoch
-        val lastNotifiedExposure = prefs.getLastNotifiedExposure()
-        if (lastExposure != null && lastExposure != lastNotifiedExposure) {
+        importLegacyExopsures()
+        db.dao().deleteOld()
+        db.dao().insert(getDailySummariesFromApi(true).map {
+            DailySummaryEntity(
+                daysSinceEpoch = it.daysSinceEpoch,
+                maximumScore = it.summaryData.maximumScore,
+                scoreSum = it.summaryData.scoreSum,
+                weightenedDurationSum = it.summaryData.weightedDurationSum,
+                importTimestamp = System.currentTimeMillis(),
+                notified = false,
+                accepted = false
+            )
+        })
+        val latestExposure = db.dao().getLatest().firstOrNull()?.daysSinceEpoch
+        val lastNotifiedExposure = db.dao().getLastNotified().firstOrNull()?.daysSinceEpoch
+        if (latestExposure != null && latestExposure != lastNotifiedExposure) {
             LocalNotificationsHelper.showRiskyExposureNotification(context)
-            prefs.setLastNotifiedExposure(lastExposure)
+            db.dao().markAsNotified()
             firebaseFunctionsRepository.registerNotification()
         } else {
-            L.i("Not showing notification, lastExposure=$lastExposure, lastNotifiedExposure=$lastNotifiedExposure")
+            L.i("Not showing notification, lastExposure=$latestExposure, lastNotifiedExposure=$lastNotifiedExposure")
+        }
+    }
+
+    //TODO: Remove in late november 2020
```

Comment:
Pokud se to má vypnout v listopadu, nechceme to řídit feature flagem z Remote Configu?

Reply:
imho není potřeba, za 14 dní až zmizí současná importovaná setkání, tak ten kód v dalších updatech úplně smažeme